src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -9739,15 +9812,210 @@ public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y)
         /// <summary>x / y</summary>
         internal readonly struct DivideOperator<T> : IBinaryOperator<T> where T : IDivisionOperators<T, T, T>
         {
+            public static bool Vectorizable => true;
             public static T Invoke(T x, T y) => x / y;
             public static Vector128<T> Invoke(Vector128<T> x, Vector128<T> y) => x / y;
             public static Vector256<T> Invoke(Vector256<T> x, Vector256<T> y) => x / y;
             public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y) => x / y;
         }
 
+        /// <summary>y / x</summary>
+        internal readonly struct InvertedDivideOperator<T> : IBinaryOperator<T> where T : IDivisionOperators<T, T, T>
```

Comment:
This is just so we don't need `InvokeScalarSpanIntoSpan`, right?

Reply:
Yup.